tfx/utils/io_utils.py

```diff
@@ -157,6 +157,11 @@ def generate_fingerprint(split_name: Text, file_pattern: Text) -> Text:
   return 'split:%s,num_files:%d,total_bytes:%d,xor_checksum:%d,sum_checksum:%d' % (
       split_name, len(files), total_bytes, xor_checksum, sum_checksum)
 
+def read_string_file(file_name: Text) -> Text:
+  """Reads a string from a file."""
+  if not tf.io.gfile.exists(file_name):
+    raise FileNotFoundError("{} does not exist".format(file_name))
```

Comment:
Can you change this line into following?
```
if six.PY2:
  raise OSError(msg)
else:
  raise FileNotFoundError(msg)  # pylint: disable=undefined-variable
```
Because `FileNotFoundError` doesn't exist in Python2, it doesn't run in Python2...